Small tweaks to all command aliases (#26403)
akka/akka: CONTRIBUTING.md

PR_TARGET_BRANCH=origin/example sbt validatePullRequest
```

> If you dont want to run all tests and just want to check that everything is formatted/mima passes there

Reviewer: ```suggestion
If you have already run all tests and now just need to check that everything is formatted and or mima passes there
```
